Userland/Applications/TextEditor/main.cpp

```diff
@@ -104,13 +107,12 @@ int main(int argc, char** argv)
     if (file_to_edit) {
         // A file name was passed, parse any possible line and column numbers included.
         FileArgument parsed_argument(file_to_edit);
-        auto absolute_path = Core::File::real_path_for(parsed_argument.filename());
-        auto file = Core::File::open(absolute_path, Core::OpenMode::ReadWrite);
+        auto file = Core::File::open(file_to_edit_full_path, Core::OpenMode::ReadOnly);
 
         if (file.is_error())
             return 1;
```

Comment:
I think this would be a nice addition as it relies on the change to `file_to_edit_full_path` :)

```cpp
#include <LibGUI/MessageBox.h>

// ...

if (file.is_error()) {
    GUI::MessageBox::show_error(window, String::formatted("Opening \"{}\" failed: {}", file_to_edit_full_path, file.error()));
    return 1;
}
```

Reply:
Good idea! Added. :)